extensions/interactions/ItemSelectionInput/directives/oppia-short-response-item-selection-input.component.ts

```diff
@@ -45,11 +45,10 @@ export class ShortResponseItemSelectionInputComponent implements OnInit {
     ) as string[];
     const choices = this.htmlEscaperService.escapedJsonToObj(
       this.choices
-    ) as { _html: string; _contentId: string }[];
+    ) as string[];
```

Comment:
Are you sure about this change? I thought the customization args would include contentIDs because of the need for translation, so I'm wondering if it was working before.

Reply:
Yes, that is the case for the function `getAnswerHtml()`, where the choices array is of the type `{_html: string; _contentId: string}[]`.

However, the display-solution-modal makes use of the `getShortAnswerHtml()` function, where each object in the choices array is a single string (only the `_html` property is included) instead of an object composed of both `_html` and `_contentId`.

In fact, that is pretty much the only dfference between `getAnswerHtml()` and `getShortAnswerHtml()`.

(See lines 202 and 203 in this image - you can see the function directly maps the `_html` property of each choice object, as opposed to mapping the complete choice object -- which is what the `getAnswerHtml()` function does):

![image](https://user-images.githubusercontent.com/78612244/158197763-8cf4b45f-8ea4-4421-9cc5-e9c25ce23c82.png)

Reply:
As for why/how it was working before, I'm guess it is because solutions weren't enabled for ItemSelectionInput, which is where the `getShortAnswerHtml()` function is used - meaning the function never got called. (Note that in other places concerning this interaction, `getAnswerHtml()` is used).